README.md

```diff
@@ -46,8 +46,7 @@ obs.RequestName({args}, callback(err, data)) returns Promise
 obs.requestName({args}, callback(err, data)) returns Promise
 
 // The following are additional supported requests.
-obs.connect({ address: 'address' }, callback(err, data)) returns Promise
-obs.authenticate({ password: 'password' }, callback(err, data)) returns Promise
+obs.connect({ address: 'address', password: 'password' }, callback(err, data)) returns Promise
```

Comment:
Technically an unauth'd connection remains open, is it preferable to create a new WebSocket internally as opposed to sending just `_authenticate`?